backend/handler/metadata/igdb_handler.py

```diff
@@ -101,6 +108,9 @@ def extract_metadata_from_igdb_rom(rom: dict) -> IGDBMetadata:
                 IGDBMetadataPlatform(igdb_id=p.get("id", ""), name=p.get("name", ""))
                 for p in rom.get("platforms", [])
             ],
+            "age_ratings": [
```

Comment:
I think we should consider the scenario where IGDB adds new age ratings, to not fail until `IGDB_AGE_RATINGS` is updated.

```python
"age_ratings": [
    IGDB_AGE_RATINGS[r["rating"]]
    for r in rom.get("age_ratings", [])
    if r["rating"] in IGDB_AGE_RATINGS
],
```